docs/scripts_md/DCMSUM.md

```diff
@@ -37,6 +37,22 @@ INPUTS:
 
 RETURNS: 1 on success
 
+### is\_study\_unique($dbh, $update, $Archivemd5)
+
+Verifies that the DICOM study is already registered in the `tarchive` tables
+using the StudyUID field of the DICOM files. If the study is already present in the
+`tarchive` tables but `-clobber` was not when running &lt;dicomTar.pl> or that we are
```

Comment:
```suggestion
`tarchive` tables but `-clobber` was not when running C<dicomTar.pl> or that we are
```

Reply:
see comment above about the use of the `C<>` tag in the perl scripts of LORIS-MRI

Reply:
@cmadjar  Just wasn't sure if it require markup or it was intended not to have markup.

Is there a rule as to when you markup or not?

Reply:
@PapillonMcGill oups sorry. I did not properly understand your earlier comment. There should indeed have been a C<> around the script names. Corrected in the next commit.
Thanks a lot!